tests/unit/wmg_processing/test_marker_gene_cube_creation.py

```diff
@@ -0,0 +1,33 @@
+from backend.wmg.pipeline.summary_cubes.marker_genes import create_marker_genes_cube
+import unittest
+from tests.unit.backend.wmg.fixtures import FIXTURES_ROOT
+import os
+import tiledb
+import shutil
+
+TEST_SNAPSHOT = "test-fmg-snapshot"
+
+# creates the marker gene cube de novo and compares to the existing fixture
+
+
+class MarkerGeneCubeCreationTest(unittest.TestCase):
```

Comment:
This expression_summary_fmg cube is implicitly being tested by this, but it may be worth having a similar test that asserts it is being built correctly. At some point the system will depend upon that data artifact directly, right? So its existence will no longer be an "implementation detail" of this marker gene cube. If you agree, it's fair to take that on as another story; but maybe easy enough to do now.

Reply:
We currently do not have any tests that `expression_summary_fmg` is being built correctly, as we don't have any tests for `expression_summary` or `cell_counts`, either. Adding tests just for `expression_summary_fmg` does not make sense as we may as well add tests for `expression_summary` and `cell_counts`, but that seems like it makes more sense to do as a separate story.

Reply:
I've created an issue to _figure out_ what remaining tests should be written, which should cover any missing tests from this effort. However, I would recommend adding a test stub for now to capture what you think _should_ be tested (it could just be a single unimplemented test).